ang/civicase/case/details/directives/case-details-header.directive.js

```diff
@@ -5,7 +5,63 @@
     return {
       replace: true,
       restrict: 'E',
-      templateUrl: '~/civicase/case/details/directives/case-details-header.directive.html'
+      templateUrl: '~/civicase/case/details/directives/case-details-header.directive.html',
+      controller: 'civicaseCaseDetailsHeaderController'
     };
   });
+
+  module.controller('civicaseCaseDetailsHeaderController', civicaseCaseDetailsHeaderController);
+
+  /**
+   *
+   * @param {*} $scope scope object of the controller
+   * @param {*} CaseActions case action service
+   * @param {*} WebformsCaseAction webform case action service
+   * @param {*} GoToWebformCaseAction go to webform case action service
+   * @param {*} webformsList configuration for webforms list
+   */
+  function civicaseCaseDetailsHeaderController ($scope, CaseActions,
+    WebformsCaseAction, GoToWebformCaseAction, webformsList) {
+    $scope.webformsAction = CaseActions.findByActionName('Webforms');
+    $scope.isGoToWebformAllowed = isGoToWebformAllowed;
+    $scope.openWebform = openWebform;
+    $scope.getWebformDropdownButtonLabel = getWebformDropdownButtonLabel;
+
+    (function init () {
+      $scope.$watch('item', itemWatcher);
+    })();
+
+    /**
+     * @returns {string} label for webform dropdown button
+     */
+    function getWebformDropdownButtonLabel () {
+      return webformsList.buttonLabel;
+    }
+
+    /**
+     * @param {object} action action object
+     */
+    function openWebform (action) {
+      GoToWebformCaseAction.doAction([$scope.item], action);
+    }
+
+    /**
+     * @param {object} action action object
+     * @returns {object} is go to webform action allowed for the sent action
+     */
+    function isGoToWebformAllowed (action) {
```

Comment:
use a verb for the function's name

Reply:
I am not sure about this point, having function names with `isXYZ` is a very common pattern that we use throughout all code. 
For example `isEmpty`, `isAllowed` etc. Lets have a quick chat.

Reply:
it's ok then